src/preview/send.ts

```diff
@@ -2,6 +2,7 @@ import express from 'express';
 import { send as sendMail } from '../helpers/mail';
 import { rpcSuccess, rpcError } from '../helpers/utils';
 import { sha256, buildMessage } from './utils';
+import { queueScheduler } from '../queues';
 
 const router = express.Router();
 const AUTH_TOKEN_HASH = 'cd372fb85148700fa88095e3492d3f9f5beb43e555e5ff26d95f5a6adc36f8e6';
```

Comment:
Hide it from what ? 

The most optimal way will be to have it completely extracted, and have its own process, like here: https://github.com/snapshot-labs/snapshot-webhook/pull/54/files#diff-d64beacb5711e9646c8e8e25cf23f283358d86c7f5f50b6f3aaf18044e356942

This will come later, in another optimization PR

Reply:
yes, that's what I meant. Because it is a sensitive token that is exposed in the public repo. It means that everyone outside can use this for personal needs.